app/src/main/res/layout/fragment_currency_details.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.main.CurrencyDetailsFragment">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I suggest you make constraint layout your root view and remove the unused framelayout :D having nested viewgroups can impact in the UI rendering.